core/util/makeConfig.js

```diff
@@ -43,15 +43,10 @@ function loadProjectConfig (command, options, config) {
       }
       userConfig = options.config;
     } else if (config.backstopConfigFileName) {
-      try {
-        // Remove from cache config content
-        +delete require.cache[require.resolve(config.backstopConfigFileName)];
-        console.log('Loading config: ', config.backstopConfigFileName, '\n');
-        userConfig = require(config.backstopConfigFileName);
-      } catch (e) {
-        console.error('Error ' + e);
```

Comment:
BTW, better way to log an error is `console.error(e.stack)`. Since the stack is already preceded by the error's `name`, you don't want to print `Error Error: ...`.